### PR TITLE
image handling: add cleanup note

### PR DESCRIPTION
**Pull request by MrTango**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ pyvenv.cfg
 
 # symlinked from submodule
 docs/volto
+/lib64
```
Comment (stevepiercy) [inline .gitignore]: This is not symlinked from submodule. Is this from a virtual environment on arch64? If so, then it should go under the `#Dependencies` heading.

File: docs/classic-ui/images.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ It can be resolved as follows:
 
 This is useful for caching URLs in Varnish or the browser.
 In case the uploaded image or scale definitions have changed, they will be saved again under a different UID.
-This changes the URL and forces either the browser, or a cache proxy such as Varnish, to fetch it again.
+This changes the URL and forces either the browser, or a cache proxy such as Varnish, to fetch it again. When a scale has changed, the old stored entry in the annotation will be deleted after 24h.
```
Comment (stevepiercy) [inline docs/classic-ui/images.md]:
```suggestion
This changes the URL and forces either the browser, or a cache proxy such as Varnish, to fetch it again.
When a scale has changed, the old stored entry in the annotation will be deleted after 24 hours.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Question on .gitignore item.
One sentence per line.
Spell out "hours".

File: docs/classic-ui/images.md [hunk 3]
```diff
@@ -70,6 +70,8 @@ It can be resolved as follows:
 This is useful for caching URLs in Varnish or the browser.
 In case the uploaded image or scale definitions have changed, they will be saved again under a different UID.
 This changes the URL and forces either the browser, or a cache proxy such as Varnish, to fetch it again.
+When a scale has changed, the old stored entry in the annotation will be deleted after 24 hours.
+If one changes an image which is used in multiple pages, the updated versions will only be shown after a restart of the Plone instance, saving the pages again or after 24h.
```
Comment (stevepiercy) [inline docs/classic-ui/images.md]: Use the Oxford comma, and spell out hours.

```suggestion
If one changes an image which is used in multiple pages, the updated versions will only be shown after a restart of the Plone instance, saving the pages again, or after 24 hours.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by mauritsvanrees — APPROVED** [review]

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
I think the failing CI is because we need to update submodule/volto to point to its master branch. I'm going to merge this, and if it doesn't build, I will update the submodule.

**Pull request merged**